addons/debugger/timing/HeatmapManager.js

```diff
@@ -0,0 +1,170 @@
+function valueToHeatmapColor(value, min = 0, max = 1) {
+  const clampedValue = Math.max(min, Math.min(max, value));
+  const normalizedValue = (clampedValue - min) / (max - min);
```

Comment:
This will be NaN if `max - min` is zero, which causes everything to be black if the slider is set to zero:
<img width="212" height="218" alt="Image" src="https://github.com/user-attachments/assets/7a35e0b4-09a5-44c5-bbc7-a41d2e70d6d9" />